readme.md

```diff
@@ -1,6 +1,6 @@
 # into-stream [![Build Status](https://travis-ci.org/sindresorhus/into-stream.svg?branch=master)](https://travis-ci.org/sindresorhus/into-stream)
 
-> Convert a string/promise/array/iterable/buffer/typedarray/arraybuffer/object into a stream
+> Convert a string/promise/array/iterable/asynciterable/buffer/typedarray/arraybuffer/object into a stream
```

Comment:
I was unsure of which casing to use here...

Reply:
This is fine